k8s/gcp-pubsub/turbinia-server.yaml

```diff
@@ -17,6 +17,13 @@ spec:
       labels:
         app: turbinia-server
     spec:
+      initContainers:
+        - name: init-filestore
+          image: busybox:1.28
+          command: ['sh', '-c', 'chmod go+w /mnt/output']
```

Comment:
Is this something we should handle in the Dockerfile (same with below for the worker)?

Reply:
The only issue I can for see is if the User wants to configure output in Kubernetes yaml to something else so would just be another place to change it else can also add a warning to user around permissions

Reply:
Makes sense, I think it's probably fine here then.